tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/package-info.java

```diff
@@ -105,8 +104,8 @@ private Documentation() {
 	 * In case the module name is {@code package.json}, see {@link #PACKAGE_JSON}.
 	 * <p>
 	 * In case the string starts with the reserved string
-	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#NODE_MODULES NODE_MODULES}, see
-	 * {@link #PROJECT_NODE_MODULES} or {@link #WORKSPACE_NODE_MODULES}.
+	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#CFG_NODE_MODULES NODE_MODULES}, see
+	 * {@link #CFG_PROJECT_NODE_MODULES} or {@link #WORKSPACE_NODE_MODULES}.
```

Comment:
It is strange that (1) `CFG_PROJECT_NODE_MODULES ` was renamed but it hadn't the same name as a CFG...-variable, and that (2) `WORKSPACE_NODE_MODULES` was not renamed.